app/parse.py

```diff
@@ -1,9 +1,25 @@
-from dataclasses import dataclass
+import csv
+import time
+from dataclasses import dataclass, fields, astuple
+from typing import List
 from urllib.parse import urljoin
 
+from bs4 import BeautifulSoup, Tag
+from selenium import webdriver
+from selenium.webdriver.chrome.webdriver import WebDriver
+from selenium.webdriver.common.by import By
 
 BASE_URL = "https://webscraper.io/"
-HOME_URL = urljoin(BASE_URL, "test-sites/e-commerce/more/")
+URLS = {
+    "home": urljoin(BASE_URL, "test-sites/e-commerce/more"),
+    "computers": urljoin(BASE_URL, "test-sites/e-commerce/more/computers"),
+    "laptops": urljoin(BASE_URL,
+                       "test-sites/e-commerce/more/computers/laptops"),
+    "tablets": urljoin(BASE_URL,
+                       "test-sites/e-commerce/more/computers/tablets"),
+    "phones": urljoin(BASE_URL, "test-sites/e-commerce/more/phones"),
+    "touch": urljoin(BASE_URL, "test-sites/e-commerce/more/phones/touch"),
+}
```

Comment:
all the links above have the same part "test-sites/e-commerce/more/"
simplify your code